src/Decompiler/Scanning/BackwardSlicer.cs

```diff
@@ -733,6 +795,39 @@ public SlicerResult VisitBinaryExpression(BinaryExpression binExp, BackwardSlice
             return se;
         }
 
+        /// <summary>
+        /// If all necessary conditions for a bounded indirect jump have been met, we can
+        /// set the jump table bounds and proceed.
+        /// </summary>
+        private SlicerResult TryFoundAllConditions(
+            BinaryExpression subtraction, 
+            StorageDomain domLeft, 
+            Expression indexCandidate, 
+            BackwardSlicerContext ctx)
```

Comment:
Is it unused method? If so should be removed

Reply:
Removed